src/main/java/frc/robot/OI.java

```diff
@@ -30,17 +33,15 @@ public OI() {
 		lowGearHold = driverControl.leftBumper;
 		highGearHold = driverControl.rightBumper;
 
-		operatorLeft = operatorControl.leftBumper;
-		operatorRight = operatorControl.rightBumper;
-		operatorLeft.whenPressed(new HatchGrabberUp());
-		operatorRight.whenPressed(new HatchGrabberDown());
-		
-		modeSwitch = operatorControl.select;
-		modeSwitch.whenPressed(new ModeSwitch(0));
```

Comment:
Did you guys get rid of mode switching for operator controls?  I think it is good to do that since it is one more thing to remember in the stress of competition.